src/components/CreateProfile/WorkInformation/WorkInformation.tsx

```diff
@@ -77,23 +73,23 @@ const WorkInformation = () => {
           Employment type
           <CheckboxInput
             id="fulltime"
-            label="Full-time"
-            checked={values.employment.includes('Full-time')}
-            onChange={() => handleEmploymentType('Full-time')}
+            label="FULL_TIME"
+            checked={values.employment === 'FULL_TIME'}
+            onChange={() => handleEmploymentType('FULL_TIME')}
             name="fulltime"
           />
           <CheckboxInput
             id="parttime"
-            label="Part-time"
-            checked={values.employment.includes('Part-time')}
-            onChange={() => handleEmploymentType('Part-time')}
+            label="PART_TIME"
+            checked={values.employment === 'PART_TIME'}
+            onChange={() => handleEmploymentType('PART_TIME')}
             name="parttime"
           />
           <CheckboxInput
             id="contract"
-            label="Contract"
-            checked={values.employment.includes('Contract')}
-            onChange={() => handleEmploymentType('Contract')}
+            label="CONTRACT"
```

Comment:
**nitpick:** This should be a `const` or enum to prevent duplicating this string